e107_plugins/forum/forum_post.php

```diff
@@ -802,8 +812,17 @@ private function forumSelect($name, $curVal=null, $opts=null)
 	 */
 	private function renderFormMove()
 	{
+		if (isset($_POST['forum_move']))
+		{
+			// Forum just moved. No need to display the forum move form again.
+			return;
+		}
+
 		if(!deftrue('MODERATOR'))
 		{
+			// todo: LAN for new error message
+			$mes = e107::getMessage();
+			echo $mes->addWarning('You do not have access to this area!')->render();
```

Comment:
Please use `LAN_NO_PERMISSIONS` (e107_languages/English/English.php

```suggestion
			echo $mes->addWarning(LAN_NO_PERMISSIONS)->render();
```